.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,17 @@
-## Proposed Changes
+[Optional] Fixes "Description of the issue" or $linkToTheIssue
 
-  -
-  -
-  -
+[Optional] RelNote [Section/Subsection] Adds a feature
```

Comment:
@m-sasha, in [your new PR](https://github.com/JetBrains/compose-multiplatform-core/pull/1271) we have RelNote/Issues fixed as sections.

Will it be okay if they will be just lines as in this template? If not, let's discuss

Reply:
I don't mind either way.

Reply:
Let's then use single-line.

It was chosen because it is compact, and it is nearby "Fixes" which will be used if no RelNote provided

Reply:
We can have issues if we need to write large release notes. But let's start with the current option and change later if needed.

Reply:
After looking at the final variant, it may be controversial 😅.

@m-sasha, @MatkovIvan, what do you prefer more?

| 1|  2 | 3 |
:-------------------------:|:-------------------------:|:-------------------------:
|![image](https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/f853d03e-c0e7-4f63-a948-c0983ab50024)|![image](https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/e4a4fed0-8ae3-44ab-884b-2545ccd8cc40)|![image](https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/fe57099d-a4c0-44ab-b3b5-3761f19ab792)|

Reply:
I wouldn't lead with the relnote, so I prefer 1 or 2

Reply:
I also prefer too. And maybe we call it "Release Notes"?

Reply:
Added as a section, fully reformatted